parsec/core/gui/instance_widget.py

```diff
@@ -47,6 +47,24 @@ async def _do_run_core(config, device, qt_on_ready):
                 await trio.sleep_forever()
 
 
+def check_macfuse_version() -> bool:
+    import os
+
+    macfuse_plist_path = "/Library/Filesystems/macfuse.fs/Contents/Info.plist"
+    if not os.path.exists(macfuse_plist_path):
+        # osxfuse installed, not macfuse
+        return False
+
+    import plistlib
+    import requests
+
+    local_version = plistlib.readPlist(macfuse_plist_path)["CFBundleVersion"]
+    remote_version = requests.get("https://api.github.com/repos/osxfuse/osxfuse/releases/latest")
```

Comment:
this is going to make the GUI freeze during the whole query so we should avoid this (typically using a job)